javascript/tests/__snapshots__/render.spec.jsx.snap

```diff
@@ -1,20 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`renders correctly 1`] = `
-<svg
-  style={
-    {
-      "display": "block",
-      "fontFamily": "arial",
-      "sansSerif": true,
-      "userSelect": "none",
-    }
-  }
->
-  <style
-    dangerouslySetInnerHTML={
-      {
-        "__html": "
+exports[`AdditiveForceVisualizer renders correctly 1`] = `
```

Comment:
I updated this, since https://github.com/shap/shap/pull/3941 breaks this test. The true tests IMO are the mpl comparisons in tests/plots